test/testdata/fixtures/single-dc-multi-cluster-medusa/k8ssandra.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cluster2-medusa
+data:
+  password: MjdrdU5adHQ3Q1YxYjJCWlZuN2c=
+  username: Y2x1c3RlcjItbWVkdXNh
+type: Opaque
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cluster2-superuser
+data:
+  password: dGVzdGVycw==
+  username: dGVzdDJAdGVzdC5jb20=
+type: Opaque
```

Comment:
I had to create those secrets because using default secrets seems to create a race condition where they will be deleted upon deletion of the K8ssandraCluster objects before the cassdc objects are deleted, but then cass-operator complains that the secrets are missing and cannot run its finalizers.